google/cloud/sql/connector/InstanceConnectionManager.py

```diff
@@ -351,6 +350,8 @@ async def _get_instance_data(self) -> InstanceMetadata:
             containing the instances IP adresses, a string representing a PEM-encoded private key
             and a string representing a PEM-encoded certificate authority.
         """
+        if not self._priv_key or not self._pub_key:
+            self._priv_key, self._pub_key = await asyncio.wrap_future(self._keys)
```

Comment:
probably don't need to assign these to `self`:
```suggestion
priv_key, pub_key = await self._keys
```

Reply:
Done